BraveWallet/Crypto/Stores/PortfolioStore.swift

```diff
@@ -121,14 +121,12 @@ public class PortfolioStore: ObservableObject {
 
   /// Fetches the prices for a given list of symbols, giving a dictionary with the price for each symbol
   private func fetchPrices(for symbols: [String], completion: @escaping ([String: String]) -> Void) {
-    assetRatioService.price(
+    assetRatioService.priceWithIndividualRetry(
```

Comment:
this is good. iirc we currently behaves differently compared to desktop and android.